site-modules/adhoc/plans/myplan.pp

```diff
@@ -0,0 +1,9 @@
+# This is a description for my plan
+plan adhoc::myplan(
+# input parameters go here
+  TargetSpec $targets,
+) {
+
+# plan steps go here
```

Comment:
Comment at indentation level content would be. It's Puppet DSL, so two spaces in.
```suggestion
  # plan steps go here
```